### PR TITLE
Fleury Algo in Python added

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes PR #17
Thanks

### CODE REVIEW

File: Fleury/fleury.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            euler(it[0], G, e, viz)
+    e.append(nod)
+
+if __name__ == "__main__":
+    input_file = sys.argv[1]
+    output_file = sys.argv[2]
+
+    with open(input_file, 'r') as fin, open(output_file, 'w') as fout:
+        n, m = map(int, fin.readline().split())
+        G = [[] for _ in range(100005)]
```
Comment (iuliagroza) [inline Fleury/fleury.py:20]: There's no need to use hardcoded values when initializing data structures. Consider the upper bound of each one and find a solution for dinamically allocating them. Hint: in this case, G is an adjacency list and each dimension is maximum `n`.

File: Fleury/fleury.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+        for _ in range(m):
+            x, y = map(int, fin.readline().split())
+            G[x].append((y, _ + 1))
+            G[y].append((x, _ + 1))
+
+        e = []
+        viz = [0] * (5 * 100005)
```
Comment (iuliagroza) [inline Fleury/fleury.py:28]: ditto

File: Fleury/fleury.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+if __name__ == "__main__":
+    input_file = sys.argv[1]
+    output_file = sys.argv[2]
+
+    with open(input_file, 'r') as fin, open(output_file, 'w') as fout:
+        n, m = map(int, fin.readline().split())
+        G = [[] for _ in range(100005)]
+
+        for _ in range(m):
```
Comment (iuliagroza) [inline Fleury/fleury.py:22]: Generally, `_` serves as a throwaway variable in Python, but in your case, the variable is used, not ignored. Consider renaming it with a suggestive name.

File: Fleury/fleury.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#  This code is contributed by Riyazul555
+
+#  Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
+#  Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+
+def euler(nod, G, e, viz):
+    while G[nod]:
+        it = G[nod].pop()
```
Comment (iuliagroza) [inline Fleury/fleury.py:8]: This is C++ style for an iterator, consider using two variables for the next node and the edge.

File: Fleury/fleury.py [hunk 5]
```diff
@@ -1,14 +1,16 @@
-#  This code is contributed by Riyazul555
+# This code is contributed by Riyazul555
 
-#  Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
-#  Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+# Time Complexity  =  O(V + E)
```
Comment (iuliagroza) [inline Fleury/fleury.py:3]: No need to remove legend.

File: Fleury/fleury.py [hunk 6]
```diff
@@ -17,15 +19,14 @@ def euler(nod, G, e, viz):
 
     with open(input_file, 'r') as fin, open(output_file, 'w') as fout:
         n, m = map(int, fin.readline().split())
-        G = [[] for _ in range(100005)]
-
+        G = [[] for _ in range(n + 1)]  
         for _ in range(m):
```
Comment (iuliagroza) [inline Fleury/fleury.py:23]: Let's replace `_` with `edge`.

**Comment by Riyazul555** [comment]
Fixed @iuliagroza

**Review by iuliagroza — APPROVED** [review]

**Pull request merged**